icon-registry/icons-static/general-sparkle-single-large-disabled_x16.svg

```diff
@@ -0,0 +1,8 @@
+<svg viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path d="M11.4321 6.68896C11.5524 6.76001 11.6747 6.82831 11.8003 6.89111L13.9917 7.98682C14.0026 7.99239 14.0026 8.0076 13.9917 8.01318L11.8003 9.10889C10.6357 9.69124 9.69124 10.6357 9.10889 11.8003L8.01318 13.9917C8.0076 14.0026 7.99239 14.0026 7.98682 13.9917L6.89111 11.8003C6.82831 11.6747 6.76001 11.5524 6.68896 11.4321L11.4321 6.68896ZM8.01318 2.0083L9.10889 4.19971C9.17169 4.3253 9.23999 4.44758 9.31104 4.56787L4.56787 9.31104C4.44758 9.23999 4.3253 9.17169 4.19971 9.10889L2.0083 8.01318C1.9974 8.0076 1.99739 7.9924 2.0083 7.98682L4.19971 6.89111C5.36432 6.30876 6.30876 5.36432 6.89111 4.19971L7.98682 2.0083C7.9924 1.99739 8.0076 1.9974 8.01318 2.0083Z" fill="currentColor" class="icon-light"/>
+  <path d="M12.2485 5.99628L14.439 7.09198C15.1866 7.46618 15.1868 8.53426 14.439 8.90839L12.2485 10.0041C11.2773 10.4897 10.49 11.277 10.0044 12.2482L8.90869 14.4387C8.53468 15.1866 7.46651 15.1865 7.09229 14.4387L5.99658 12.2482C5.98295 12.221 5.96772 12.1941 5.95361 12.1672L7.41553 10.7053C7.54939 10.9141 7.6739 11.1302 7.78564 11.3537L8.00049 11.7834L8.21533 11.3537C8.89448 9.99542 9.99572 8.89417 11.354 8.21503L11.7837 8.00018L11.354 7.78534C11.1305 7.67361 10.9144 7.54907 10.7056 7.41522L12.1675 5.95331C12.1944 5.9674 12.2213 5.98266 12.2485 5.99628ZM7.09229 1.56171C7.44299 0.860329 8.40371 0.816021 8.83057 1.42987L8.90869 1.56171L10.0044 3.75214C10.0179 3.77912 10.0324 3.80552 10.0464 3.83221L8.58447 5.29413C8.4508 5.08555 8.32695 4.86987 8.21533 4.64667L8.00049 4.21698L7.78564 4.64667C7.10654 6.00487 6.00514 7.10618 4.64697 7.78534L4.21729 8.00018L4.64697 8.21503C4.87016 8.32664 5.08586 8.4505 5.29443 8.58417L3.83252 10.0461C3.80583 10.0321 3.77942 10.0176 3.75244 10.0041L1.56201 8.90839C0.813928 8.53432 0.813848 7.46606 1.56201 7.09198L3.75244 5.99628C4.72355 5.51065 5.51101 4.72328 5.99658 3.75214L7.09229 1.56171Z" fill="currentColor" class="icon-dark"/>
+  <path d="M2 14L14 2" stroke="#E45770" stroke-linecap="round"/>
+</svg>
+
```

Comment:
@emilmilanov There's a lot of line breaks at the end of this file for some reason. Typically this is something a linter would catch, so just something to look into.